app/models/workers_scaling/experiment_statistics.rb

```diff
@@ -0,0 +1,60 @@
+##
+# ExperimentStatistics class calculates various statistics about
+# experiment run and execution.
+class ExperimentStatistics
+
+  ##
+  # Number of running simulations in sequential worker
+  RUNNING_SIMULATIONS = 1
+
+  def initialize(experiment, resources_interface)
+    @experiment = experiment
+    @simulation_runs = @experiment.simulation_runs
+    @resources_interface = resources_interface
+  end
+
+  ##
+  # Returns SimulationRuns with given params
+  # params:
+  #   * opts
+  #   * cond
+  def for_simulation_runs(params = {})
```

Comment:
Hm. To jest totalny bezsens teraz. Chyba docelowo to miało jeszcze filtrować po polach żeby dostać tylko te które potrzebujemy, ale skoro tego nie ma to trzeba to wywalić.